### DISCUSSION TITLE
Is it possible when working with mongoDB to use aggregate ($ avg) function like in mongoose?

### DISCUSSION THREAD

**Discussion by Devero97** [prisma/prisma#10784] [Q&A]
Is it possible when working with mongoDB to use aggregate ($ avg) function like in mongoose? To calculate the average rating? Can't find it in the documentation
I saw it. But can he do what I want?
I have a post

```
post: {
  title: "Future devops", 
  avgrating: 5, 
  ratingOne: 4
  ratingTwo: 4 
  ratingThree: 4       
  ratingFour: 5 
  ratingFive: 4
}
```
And comments to it

```
commentsPost: [
  {
  name: "Dima", 
  ratingOne: 3,
  ratingTwo: 2,
  ratingThree: 2,      
  ratingFour: 5,
  ratingFive: 2
  },
  {
  name: "Vanya", 
  ratingOne: 3,
  ratingTwo: 2,
  ratingThree: 2,      
  ratingFour: 5,
  ratingFive: 2
  },
  {
  name: "Kirill", 
  ratingOne: 3,
  ratingTwo: 2,
  ratingThree: 2,      
  ratingFour: 5,
  ratingFive: 2
  },
  {
  name: "Luci", 
  ratingOne: 3,
  ratingTwo: 2,
  ratingThree: 2,      
  ratingFour: 5,
  ratingFive: 2
  },
  {
  name: "Vadim", 
  ratingOne: 3,
  ratingTwo: 2,
  ratingThree: 2,      
  ratingFour: 5,
  ratingFive: 2
  },
 ]
```
Can I get each rating for a post as an average?
In addition, the post has an avgrating field that changes based on the average ratings of its other fields (ratingOne, ratingTwo, ratingThree, ratingFour, ratingFive)

**Comment by Devero97** [comment]
Please answer my question!)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
